### ISSUE TITLE
Homestead Up results in apache not starting and have to reload --provision to fix

### ISSUE THREAD

**Issue by terryupton** [laravel/homestead#1047]
Please note that the Homestead issue tracker is reserved for bug reports and enhancements. We are not always able to debug Vagrant, Provider or Operating System issues, but will do our best to help. Thank you!

## Versions
- Vagrant: Vagrant 2.2.3
- Provider: Virtualbox 6.0.2r128162
- Homestead: homestead v8.0.0

## Host operating system
- macOS Mojave Version 10.14.2

## Homestead.yaml
```yaml
ip: "192.168.10.10"
memory: 2048
cpus: 2
provider: virtualbox
mariadb: true
authorize: ~/.ssh/id_rsa.pub

keys:
    - ~/.ssh/id_rsa

folders:
    - map: ~/Dropbox/Amasci/Websites
      to: /home/vagrant/sites
      type: nfs

sites:
    - map: crafttest.loc
      to: '"/home/vagrant/sites/crafttest-v3/web"'
      type: apache
      php: "7.2"

    - map: craftbp.loc
      to: '"/home/vagrant/sites/craftbp/web"'
      type: apache
      php: "7.2"

    - map: plowmancraven.loc
      to: '"/home/vagrant/sites/Plowman Craven V4/plowmancraven.co.uk"'
      type: apache
      php: "7.1"

    - map: cladwinds.loc
      to: '"/home/vagrant/sites/Cladwinds Limited V2/cladwinds.co.uk"'
      type: apache
      php: "7.1"

    - map: woburnmosaic.loc
      to: '"/home/vagrant/sites/Woburn Mosaic/woburnmosaic.co.uk"'
      type: apache
      php: "7.2"

    - map: battlefieldjourney.loc
      to: '"/home/vagrant/sites/Battlefield Journey V3/battlefieldjourney.com"'
      type: apache
      php: "7.1"

    - map: amasci.loc
      to: '"/home/vagrant/sites/Amasci v4/amasci.co.uk"'
      type: apache
      php: "7.1"

    - map: swh.loc
      to: '"/home/vagrant/sites/Scott White & Hookins V4/swh.co.uk"'
      type: apache
      php: "7.1"

    - map: martinwilliamsart.loc
      to: '"/home/vagrant/sites/Martin Williams V2/martinwilliamsart.co.uk"'
      type: apache
      php: "7.1"

    - map: deluxedevelopers.loc
      to: '"/home/vagrant/sites/Deluxe Developers Limited/deluxedevelopers.co.uk"'
      type: apache
      php: "7.1"

    - map: hunsburyhottubs.loc
      to: '"/home/vagrant/sites/Hunsbury Hot Tubs V2/hunsburyhottubs.co.uk/web"'
      type: apache
      php: "7.2"

    - map: i-swh.loc
      to: '"/home/vagrant/sites/Scott White & Hookins Intranet/i-swh.co.uk"'
      type: apache
      php: "7.1"

    - map: ecoknittowels.loc
      to: '"/home/vagrant/sites/EcoKnit/ecoknittowels.com"'
      type: apache
      php: "7.1"

    - map: regencyexectravel.loc
      to: '"/home/vagrant/sites/Regency Executive Travel Limited/regencyexectravel.co.uk"'
      type: apache
      php: "7.1"

    - map: boxxcomms.loc
      to: '"/home/vagrant/sites/Boxx Communications Limited V2/boxxcomms.co.uk"'
      type: apache
      php: "7.1"

    - map: dengrove.loc
      to: '"/home/vagrant/sites/Dengrove V3/dengrove.com"'
      type: apache
      php: "7.1"

    - map: join-the-quest.loc
      to: '"/home/vagrant/sites/Join The Quest V2/join-the-quest.co.uk"'
      type: apache
      php: "7.1"

    - map: geomatrix.loc
      to: '"/home/vagrant/sites/Geomatrix Earth Science Limited V2/geomatrix.co.uk"'
      type: apache
      php: "7.1"

    - map: supershoes.loc
      to: '"/home/vagrant/sites/Supershoes/supershoes.org.uk"'
      type: apache
      php: "7.1"

    - map: grommets.loc
      to: '"/home/vagrant/sites/Grommets Limited V2/grommets.co.uk"'
      type: apache
      php: "7.1"

    - map: romanrangersfc.loc
      to: '"/home/vagrant/sites/Roman Rangers FC/romanrangersfc.co.uk"'
      type: apache
      php: "7.2"

    - map: plowmancraven.loc.com
      to: '"/home/vagrant/sites/Plowman Craven V4/plowmancraven.co.uk"'
      type: apache
      php: "7.1"

    - map: cvpal.loc
      to: '"/home/vagrant/sites/CVpal V3/cvpal.co.uk"'
      type: apache
      php: "7.1"

    - map: riaflex.loc
      to: '"/home/vagrant/sites/Riaflex V3/riaflex.co.uk"'
      type: apache
      php: "7.1"

    - map: thembrsite.loc
      to: '"/home/vagrant/sites/The MBR Site V2/thembrsite.com"'
      type: apache
      php: "7.1"

    - map: juddwater.loc
      to: '"/home/vagrant/sites/Judd Water & Wastewater Services/juddwater.com"'
      type: apache
      php: "7.1"

    - map: davidnewens.loc
      to: '"/home/vagrant/sites/David Newens V2/davidnewens.com"'
      type: apache
      php: "7.1"

    - map: electrichearses.loc
      to: '"/home/vagrant/sites/Brahms Electric Hearses/electrichearses.co.uk"'
      type: apache
      php: "7.1"

databases:
    - amasci
    - boxxcomms
    - cladwinds
    - plowmancraven
    - cvpal
    - davidnewens
    - dengrove
    - ecoknittowels
    - geomatrix
    - grommets
    - hunsburyhottubs
    - iswh
    - jointhequest
    - juddwater
    - martinwilliamsart
    - riaflex
    - romanrangers
    - supershoes
    - swh
    - tailwind
    - thembrsite
    - electrichearses
    - woburnmosaic
    - craftbp
    - crafttest

variables:
    - key: APP_ENV
      value: local
```

## Expected behavior
Apache should start and test sites load.

## Actual behavior
Homesatead Up results in 502 bad gateway and apache doesn't start and fails to start with the following error:

## Steps to reproduce
1. Run Homestead Up
2. Try access local sites
3. Homestead SSH and run sudo apache2 status or run sudo apache2 start

## References
```
● apache2.service - The Apache HTTP Server
   Loaded: loaded (/lib/systemd/system/apache2.service; enabled; vendor preset: enabled)
  Drop-In: /lib/systemd/system/apache2.service.d
           └─apache2-systemd.conf
   Active: failed (Result: exit-code) since Sun 2019-01-27 19:37:48 UTC; 11s ago
  Process: 3147 ExecStart=/usr/sbin/apachectl start (code=exited, status=1/FAILURE)

Jan 27 19:37:48 homestead systemd[1]: Starting The Apache HTTP Server...
Jan 27 19:37:48 homestead apachectl[3147]: (98)Address already in use: AH00072: make_sock: could not bind to address [::]:80
Jan 27 19:37:48 homestead apachectl[3147]: (98)Address already in use: AH00072: make_sock: could not bind to address 0.0.0.0:80
Jan 27 19:37:48 homestead apachectl[3147]: no listening sockets available, shutting down
Jan 27 19:37:48 homestead apachectl[3147]: AH00015: Unable to open logs
Jan 27 19:37:48 homestead apachectl[3147]: Action 'start' failed.
Jan 27 19:37:48 homestead apachectl[3147]: The Apache error log may have more information.
Jan 27 19:37:48 homestead systemd[1]: apache2.service: Control process exited, code=exited status=1
Jan 27 19:37:48 homestead systemd[1]: apache2.service: Failed with result 'exit-code'.
Jan 27 19:37:48 homestead systemd[1]: Failed to start The Apache HTTP Server.
```

Only way I can solve this is homestead reload --provision

**Comment by svpernova09** [comment]
Comment out all but one of your sites, does the issue happen with 1 site instead of all of them?

Also you forgot a part:

## Vagrant destroy & up output
Provide a link to a [GitHub Gist](https://gist.github.com/) containing the complete output of `vagrant destroy && vagrant up`.
Do **NOT** paste the debug output in the issue, just paste the link to the Gist.

**Comment by browner12** [comment]
can you manually start apache either with

`sudo service apache2 start`

or

`flip`

**Comment by terryupton** [comment]
So it appears that nginx is starting up as well as apache and because nginx loads first apache is failing to load. By stopping nginx and then starting apache it seems to resolve this.

**Comment by svpernova09** [comment]
I need to dig into this more. I bet it's socket-wrench not being handled properly by the apache script.

**Comment by browner12** [comment]
yah, always a little bit of a race condition you can run into here with both Apache and Nginx installed

**Comment by svpernova09** [comment]
I'm 99% sure the problem is the newly added built in socket-wrench server is what's causing the issue. I currently don't have a good solution (but still working through one)

**Comment by svpernova09** [comment]
Should be resolved by v8.1.0
